model/src/main/java/org/projectnessie/api/params/ReferencesParams.java

```diff
@@ -44,21 +45,44 @@
   @QueryParam("fetchAdditionalInfo")
   private boolean fetchAdditionalInfo;
 
+  @Parameter(
+      description =
+          "A Common Expression Language (CEL) expression. An intro to CEL can be found at https://github.com/google/cel-spec/blob/master/doc/intro.md.\n"
+              + "Usable variables within the expression are 'entry.namespace' (string) & 'entry.contentType' (string)",
```

Comment:
Do `entry.namespace` and `entry.contentType` actually apply to references?

Reply:
Same here